### ISSUE TITLE
Add examples to SDL3/SDL_CreateWindowWithProperties?

### ISSUE THREAD

**Issue by Susko3** [libsdl-org/sdlwiki#535]
Since properties are a new concept, and creating a window is one of the first things you do in SDL, I think it would be useful to have an example of how to use it. Showing different property types, error checking, etc. I think this is best done by the core team, as they know how properties should be used.

Would it make sense to add `CategoryProperties` to `SDL_Create*WithProperties` pages?

**Comment by slouken** [comment]
I don't think we need a separate category for properties - they belong with the other functions in their category. An example for creating a window with properties is a great idea!

**Comment by icculus** [comment]
Yeah, mostly Categories map to how we have things broken up by headers. It's not really a See Also sort of thing.
